config_example.sh

```diff
@@ -783,6 +783,9 @@ set -x
 # the assisted-service validations pass.
 # export AGENT_PLATFORM_TYPE=baremetal
 
+# When set to 'oci', the CloudControllerManager field is set to External in the install-config.yaml.
+# export AGENT_PLATFORM_NAME=oci
```

Comment:
Will need to add a setting to agent/common.sh to avoid unit'ed value, e.g.
`export AGENT_PLATFORM_NAME=${AGENT_PLATFORM_NAME:-"oci"}`

Reply:
Yes, it is already added here https://github.com/openshift-metal3/dev-scripts/pull/1611/files#diff-2873b86d2f543deb97334c2e888ee7f4f0549cad18d8eb4ec0d3c9e86170031cR12